client/src/client.rs

```diff
@@ -405,20 +416,17 @@ pub trait RpcApi: Sized {
         signature: &Signature,
         message: &str,
     ) -> Result<bool> {
-        let args = [into_json(address)?, into_json(signature)?, into_json(message)?];
+        let args = [address.to_string().into(), signature.to_string().into(), into_json(message)?];
```

Comment:
Does this pattern means allocating twice?

Reply:
I'm not sure. Shouldn't be. I think `From<String> for Value` should not re-allocate.